skaffold/src/main/kotlin/com/google/container/tools/skaffold/SkaffoldConfigurationDetector.kt

```diff
@@ -66,17 +72,39 @@ class SkaffoldConfigurationDetector(val project: Project) : ProjectComponent {
             val skaffoldFiles: List<VirtualFile> =
                 SkaffoldFileService.instance.findSkaffoldFiles(project)
             if (skaffoldFiles.isNotEmpty()) {
-                val skaffoldRunConfigList: List<RunConfiguration> =
-                    getRunManager(project)
-                        .allConfigurationsList.filter { it is AbstractSkaffoldRunConfiguration }
-                if (skaffoldRunConfigList.isEmpty()) {
+                if (!hasExistingSkaffoldConfigurations()) {
                     // existing Skaffold config files, but no skaffold configurations, prompt
                     showPromptForSkaffoldConfigurations(project, skaffoldFiles[0])
                 }
             }
         }
+
+        // add VFS listener to ensure we track YAML file changes when no Skaffold configurations
```

Comment:
perhaps move this comment to `addVirtualFileListener` or add another one there too

Reply:
on second look. a different more generic comment on `addVirtualFileListener` seems more appropriate

Reply:
done.